hw28/index.js

```diff
@@ -0,0 +1,171 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    function calcAvg(arr) {
```

Comment:
I'd get rid of shuffling methods and functions. Let's make it static method. Get some work with its name